src/app/components/userprofile/userprofile.component.css

```diff
@@ -0,0 +1,24 @@
+.imgProfile{
```

Comment:
you are using different styles to name your css classes... I suggest to use `.image-profile`, `.credentials` and so on... here is a reference about that: http://bdavidxyz.com/blog/how-to-name-css-classes/